internal/gengapic/gengrpc.go

```diff
@@ -295,6 +295,8 @@ func (g *generator) grpcClientInit(serv *descriptorpb.ServiceDescriptorProto, se
 
 	p("// The x-goog-* metadata to be sent with each request.")
 	p("xGoogHeaders []string")
+	p("")
+	p("logger *slog.Logger")
```

Comment:
Could this and the import below be moved to `clientInit`?

Reply:
no, this is needed in by the concrete impl clients. It could also be added to that top level client, but I don't think there is a benefit.